pkg/apis/pipeline/v1/result_types.go

```diff
@@ -72,6 +72,9 @@ type TaskRunResult struct {
 	Value ResultValue `json:"value"`
 }
 
+// StepRunResult is a type alias of TaskRunResult
+type StepRunResult = TaskRunResult
```

Comment:
I don't like that we call this `StepRunResult` because we don't have a `StepRun`, but I also don't know what else we can call it -- reminds of this issue about consolidating this object: https://github.com/tektoncd/pipeline/issues/2825

Reply:
I like TaskRunStepResult as an alternative.

Reply:
Sounds good to me

We can consolidate those fields later, maybe in V2

Reply:
Yup! Changed `StepRunResult` to `TaskRunStepResult` since that is more accurate.